sw/device/tests/BUILD

```diff
@@ -382,6 +382,31 @@ opentitan_functest(
     deps = ["clkmgr_off_trans_impl"],
 )
 
+opentitan_functest(
+    name = "clkmgr_reset_frequency_test",
+    srcs = ["clkmgr_reset_frequency_test.c"],
+    cw310 = cw310_params(
+        # FIXME #13611
+        tags = ["broken"],
```

Comment:
can you remind me, is this broken for the same reason as the other test? because `ast_init_done` is not high so measurements never start?

Reply:
Yes, the failure is consistent with it.